#This message describes the motor as the component of the following:
#1. Direction of rotation
#2. PWM value
int16 dor
int16 pwm